pkg/common/types.go

```diff
@@ -120,3 +120,21 @@ const (
 	// Failed means the task has failed its execution.
 	Failed ConditionType = "Failed"
 )
+
+type OwnerReference struct {
```

Comment:
Can we reuse `metav1`? 
```go	
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"

metav1.OwnerReference
```

Reply:
We have to add `enable` which is not inside of `metav1.OwnerReference`

Reply:
got it. how about the  following struct ? 
```go
type OwnerReference struct {
     metav1.OwnerReference 
     Enable bool
}
```